audio_program_generator/apg.py

```diff
@@ -68,14 +69,22 @@
 from tqdm import tqdm
 
 
-def num_lines_in_file(filename: Path) -> int:
-    """
-    Takes text filename ; returns the number of lines in a file
-    """
-    numlines = 1
-    with open(filename, "r"):
-        numlines += 1
-    return numlines
+def parse_textfile(filename: str="") -> list:
+    def clean(input: str="") -> str:
+        cleaner = r"[^A-Za-z0-9\s;\v]"
+        clean = re.compile(cleaner, flags=re.MULTILINE | re.UNICODE)
+        return re.sub(clean, '', input)
+
+    def capture(cleaned: str="") -> list:
```

Comment:
Capture what?

Reply:
Ok,, "extract" might be a better name, bit as it is an internal function perhaps it does not matter so much :-)